meteor-app/imports/ui/service/service-context.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, createContext } from 'react'
+import ServiceItems from '/imports/api/assessments/serviceItems'
+import Services from '/imports/api/assessments/services'
+// import service from './index'
+
+console.log(ServiceItems)
+
+// import {createContainer} from "meteor/react-meteor-data"
+
+export const ServiceContext = createContext()
+
+const data = [
+  { title: 'item1', isMajMin: false },
+  { title: 'item2', isMajMin: false },
+  { title: 'item3', isMajMin: false },
+  { title: 'item4', isMajMin: false },
+  { title: 'item5', isMajMin: false },
+  { title: 'item6', isMajMin: false },
+  { title: 'item6', isMajMin: false },
+
+  { title: 'item17', isMajMin: false },
+  {
+    title: 'Major Service',
+    isMajMin: true,
+    items: [
+      { title: 'mj1', greyed: false },
+      { title: 'mj2', greyed: false },
+      { title: 'mj3', greyed: false },
+      { title: 'mj4', greyed: false },
+      { title: 'mj5', greyed: false },
+      { title: 'mj16', greyed: false },
+      { title: 'mj17', greyed: false },
+      { title: 'mj18', greyed: false }
+    ]
+  },
+  {
+    title: 'Minor Service',
+    isMajMin: true,
+    items: [
+      { title: 'mn1', greyed: false },
+      { title: 'mn2', greyed: false },
+      { title: 'mn3', greyed: false },
+      { title: 'mn4', greyed: false },
+      { title: 'mn5', greyed: false },
+      { title: 'mn6', greyed: false },
+      { title: 'mn18', greyed: false },
+      { title: 'mn19', greyed: false },
+      { title: 'mn20', greyed: false }
+    ]
+  }
+]
+
+const serviceState = {
+  data: data,
+  tags: []
+}
+
+Tracker.autorun(() => {
+  let serviceItemsHandle = Meteor.subscribe('all.serviceItems')
```

Comment:
This is the reason why your components don't work in Storybook - the context file should be 'pure' and not referencing Meteor functions. The Tracker code and subscriptions should be moved out to a 'container' component, and the data then passed in to the context (or the components)